servers/rendering/dummy/storage/texture_storage.h

```diff
@@ -151,14 +151,17 @@ class TextureStorage : public RendererTextureStorage {
 	virtual RID render_target_create() override { return RID(); }
 	virtual void render_target_free(RID p_rid) override {}
 	virtual void render_target_set_position(RID p_render_target, int p_x, int p_y) override {}
```

Comment:
I am starting to wonder whether we need this API, is this used anywhere really?

Reply:
`render_target_create` is called when a new viewport is registered with the rendering server to start creating the render target textures. `render_target_free` is called when the viewport is destroyed. So those are needed.

`render_target_set_position` is also still called from the viewport logic but I think it's only used in GLES, could be wrong, @clayjohn do you remember how this is used?

Reply:
Ah I remember now, that's right, I completed the API for render target adding missing functions for each member for when we eventually expose this in GDExtension so the change itself has nothing to do with the override. 

Github is being a bit weird with positioning these comments, so we're talking about `render_target_set/get_position` and `render_target_set/get_size` ? Yeah I have no idea if they are actually still used or if they may only be applicable to GLES. But they are settings on our viewport so it's a question if it is enough to record them there or if they indeed need to be recorded on the render target.

Reply:
I think it is likely better to keep this if we want to do the rendering directly to window optimization in OpenGL3 (or in the modern renderers). We need a position and size if we render directly to screen.